src/components/organisms/ProfileModal/ProfilePopoverContent.scss

```diff
@@ -15,7 +15,7 @@ $profile-popover--margin-top: 12px;
   position: relative;
   overflow: auto;
   padding: $spacing--md;
-  background: $black;
+  @include background-color($black);
```

Comment:
is there any advantage using @include instead of pure background there?

Reply:
yes

Reply:
NOTE: function sets the needed CSS --variables for the descendant indicator border to preclude the direct parent's paint.